Common/Statistics/PortfolioStatistics.cs

```diff
@@ -238,6 +250,8 @@ public class PortfolioStatistics
             WinRate = totalTrades == 0 ? 0 : (decimal) totalWins / totalTrades;
             LossRate = totalTrades == 0 ? 0 : (decimal) totalLosses / totalTrades;
             Expectancy = WinRate * ProfitLossRatio - LossRate;
+            StartingEquity = equity.FirstOrDefault().Value;
```

Comment:
Please review how's/where `startingCapital` set should probably be using that value as starting equity I believe

Reply:
Yup, we can use `startingCapital` since that value comes from `BacktestingSetupHandler.StartingPortfolioValue`, and that value is setup just once when the algorithm starts. See `https://github.com/QuantConnect/Lean/blob/master/Engine/Engine.cs#L225`